decoder_test.go

```diff
@@ -294,4 +294,46 @@ func TestDataMultipleAtendee(t *testing.T) {
 		t.Errorf("Atendee list should be %s", att)
 	}
 
-}
\ No newline at end of file
+}
+
+var dataMultiline = `BEGIN:VCALENDAR
+BEGIN:VEVENT
+ORGANIZER:Mailto:B@example.com
+ATTENDEE;
+ ROLE=CHAIR;
+  PARTSTAT
+   =ACCEPTED;CN=
+    BIG A
+	 :Mailto:A@example.com
+ATTENDEE;RSVP=TRUE;CUTYPE=INDIVIDUAL;
+ CN=B:Mailto:B@example.com
+ATTENDEE;RSVP=TRUE;CUTYPE=INDIVIDUAL;CN=C:Mailto:C@example.com
+DTSTAMP:19970611T193000Z
+DTSTART:19970701T190000Z
+DTEND:19970701T193000Z
+UID:calsrv.example.com-873970198738777@example.com
+SUMMARY: Pool party
+END:VEVENT
+END:VCALENDAR`
+
+func TestMulitine(t *testing.T) {
+
+	d := goics.NewDecoder(strings.NewReader(dataMultiline))
+	consumer := EventsA{}
+	err := d.Decode(&consumer)
+	if err != nil {
+		t.Error("Error decoding events")
+	}
+	if len(consumer) != 1 {
+		t.Error("Wrong size of consumer list..")
+	}
+
+	if len(consumer[0].Attendees) != 3 {
+		t.Errorf("Wrong list of atendees detectet %d", len(consumer[0].Attendees))
```

Comment:
> Wrong list of attendees detected %d

Maybe wrong "size" instead of "list" too?

Also, I tend to store the value in a local variable so the error message line doesn't have to repeat it. Makes it easier to change or copy around if you have to test different properties in the same way.

```go
if size := len(consumer[0].Attendees); size != 3 {
	t.Errorf("Wrong size of attendees detected %d", size)
```